docs/.vitepress/navbar.ts

```diff
@@ -15,6 +15,7 @@ const navbar: DefaultTheme.NavItem[] = [
                     { text: 'Smart Contracts', link: '/tutorials/smart-contracts/index' },
                     { text: 'Uniq Factories', link: '/tutorials/uniq-factories/index' },
                     { text: 'Token Swap', link: '/tutorials/token-swap/index' },
+                    { text: 'Fungitable Token', link: '/tutorials/token/index' },
```

Comment:
This will cause each link validation to fail because it will be searching for `index` file on all 3 environments. We will need to either not have this here or add a support for environment-based navbar (e.g. by setting `evn: "experimental"`

Reply:
I changed the index back to normal. I think we can consider it as a placeholder.